### PR TITLE
expand on Global Element section

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -743,7 +743,9 @@ type: Unsigned Integer
 
 description: The minimum `DocType` version an `EBML Reader` has to support to read this `EBML Document`. The value of the `DocTypeReadVersion Element` MUST be less than or equal to the value of the `DocTypeVersion Element`.
 
-## Global elements (used everywhere in the format)
+## Global Elements
+
+EBML defines these `Global Elements` which MAY be stored within any Element of an `EBML Document` as defined by their `Element Path`.
```
Comment (mbunkus) [inline specification.markdown]: Maybe rather say "within any Master Element"? Just to be extra precise.

**Review by mbunkus — DISMISSED** [review]

**Comment by ablwr** [comment]
If global elements are only possible, should you use language stronger than MAY? MAY implies "truly optional."

**Comment by robUx4** [comment]
The `MAY` refers to the `be stored` not the fact that it MUST be in an `Master Element`, if that's what you mean.

**Comment by ablwr** [comment]
Ahhh, of course, my mistake. It's still early here. 😅

**Review by mbunkus — APPROVED** [review]
LGTM with that commit from Steve.

**Pull request merged**